source/code/scxsystemlib/disk/statisticallogicaldiskenumeration.cpp

```diff
@@ -234,6 +234,12 @@ namespace SCXSystemLib
             SCXCoreLib::SCXHandle<StatisticalLogicalDiskInstance> disk = *iter;
             disk->Update();
 
+//Fix for Bug10019956
+#if defined(sun)
+        if(disk->m_fsType == L"zfs" && (disk->m_device).find(L"/") != -1 && (disk->m_device).find(L"/") != 0)
```

Comment:
By convention, when parsing files, we have (in a comment) a sample of the line we're trying to parse.

Please add that between 237 and 238, I'd like to see that before signing off please.

Also, lines 237 and 239 aren't lining up with 235, please fix indentation.